src/middlewared/middlewared/alert/source/multipaths.py

```diff
@@ -5,7 +5,7 @@ class MultipathIsNotOptimalAlertClass(AlertClass):
     category = AlertCategory.HARDWARE
     level = AlertLevel.CRITICAL
     title = "Multipath Is Not Optimal"
-    text = "Multipath %s is not optimal."
+    text = "Multipath %s is not optimal. Please check your disks cabling."
```

Comment:
s/disks/disk/ (it works for the plural here)

I think a lot of people see this from something other than cabling when I searched on the error.  Multiple GEOM devices on a single machine being seen as multipath?  I don't know.

Reply:
Oops, forgot to mention the "your" again. Suggestion:
"Please check disk cables."
That will be misleading if it is caused by something logical rather than electrical.